vcr/vcr.go

```diff
@@ -457,6 +457,27 @@ func (c *vcr) Untrust(credentialType did.URI, issuer did.URI) error {
 	return c.trustConfig.RemoveTrust(credentialType, issuer)
 }
 
+func (c *vcr) Find(conceptName string, ID did.DID) (concept.Concept, error) {
+	q, err := c.Registry().QueryFor(conceptName)
+	if err != nil {
+		return nil, err
+	}
+
+	q.AddClause(concept.Eq(concept.SubjectField, ID.String()))
+
+	vcs, err := c.Search(q)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(vcs) == 0 {
+		return nil, ErrNotFound
```

Comment:
I recall that we decided that `Find` (similar to Search) operations generally return a list and more importantly don't return an error when not found? When returning a single entry and an error when it's not found it feels more like `Resolve` or `Get` (when you don't like resolve). `ConceptResolver`?

Reply:
It's defined on the vcr what makes naming a bit trickier

Reply:
Named it to `Get`